.github/workflows/add_label_from_diff.yaml

```diff
@@ -0,0 +1,39 @@
+name: Autolabel PRs
+
+on:
+  pull_request:
+    types: [opened]
+  workflow_dispatch:
```

Comment:
@grunweg I updated the workflow, so now it only triggers once when the PR is opened, but there is also the option to dispatch it manually from the github UI

Reply:
There is one catch with this:

Since the workflow only triggers once, one could add a new folder in say the second commit to the PR. This might lead to a passing CI but a new PR created will then complain that not all of Mathlib is covered by the labels....

Reply:
Do we need a test that all of mathlib is covered by labels? If so, could we run it say just once a week or something? I'd really like to avoid forcing people to have to solve unrelated CI issues in their PRs if something sneaks through in someone else's PR...

Reply:
I ended up that this only triggers a warning. So there might be an unexpected warning on future PRs, but at least it should not cause anything to fail.